### PR TITLE
Initial 0.10.0.4 upgrade

### PR DESCRIPTION
**Pull request by Wayonb**
The initial 0.10.0.4 release with preset updated.  
Voting will not work until TS sdk is updated with the latest voting key changes.

### CODE REVIEW

File: config/node/resources/config-finalization.properties.mustache [hunk 1]
```diff
@@ -11,4 +12,5 @@ messageSynchronizationMaxResponseSize = {{{messageSynchronizationMaxResponseSize
 
 maxHashesPerPoint = {{{toAmount maxHashesPerPoint}}}
 prevoteBlocksMultiple = {{{toAmount prevoteBlocksMultiple}}}
-votingKeyDilution = {{{votingKeyDilution}}}
+
+unfinalizedBlocksDuration = 0m
```
Comment (fboucquez) [inline config/node/resources/config-finalization.properties.mustache:16]: move value to shared.yml

File: config/node/resources/config-messaging.properties.mustache [hunk 2]
```diff
@@ -1,3 +1,5 @@
 [messaging]
 
 subscriberPort = {{{subscriberPort}}}
+
+listenInterface = 0.0.0.0
```
Comment (fboucquez) [inline config/node/resources/config-messaging.properties.mustache:5]: move value to shared.yml

File: config/node/resources/config-node.properties.mustache [hunk 3]
```diff
@@ -43,15 +43,19 @@ enableDispatcherInputAuditing = {{{enableDispatcherInputAuditing}}}
 maxCacheDatabaseWriteBatchSize = {{{maxCacheDatabaseWriteBatchSize}}}
 maxTrackedNodes = {{{toAmount maxTrackedNodes}}}
 
+minPartnerNodeVersion = {{{minPartnerNodeVersion}}}
+maxPartnerNodeVersion = {{{maxPartnerNodeVersion}}}
+
 # all hosts are trusted when list is empty
 trustedHosts = {{{trustedHosts}}}
 localNetworks = {{{localNetworks}}}
+listenInterface = 0.0.0.0
```
Comment (fboucquez) [inline config/node/resources/config-node.properties.mustache:52]: move value to shared.yml

File: config/node/resources/config-node.properties.mustache [hunk 4]
```diff
@@ -43,15 +43,19 @@ enableDispatcherInputAuditing = {{{enableDispatcherInputAuditing}}}
...
 
 [localnode]
 
 host = {{{host}}}
 friendlyName = {{{friendlyName}}}
-version = 0
+version = 0.10.0.4
```
Comment (fboucquez) [inline config/node/resources/config-node.properties.mustache:58]: move value to shared.yml

**Review by fboucquez — CHANGES_REQUESTED** [review]

**Comment by fboucquez** [comment]
Thanks @Wayonb , ideally we should have all the values in the shared.yml by default. If a value is variable enough to be in a property file, it should be variable enough to be in shared.yml so node admin can change it with custom presets.

File: config/node/resources/config-messaging.properties.mustache (hunk 2 above)
Comment (fboucquez) [reply config/node/resources/config-messaging.properties.mustache:5]: unsure if this is shared value with the one in config-node or not. If this value could be different, add a prefix to listenInterface like `messaginListenInterface` and `nodeListenInterface`

**Comment by Wayonb** [comment]
@fboucquez Thanks for the feedback and fixes.  I fixed the tests :)

**Pull request merged**